internal/cli/render.go

```diff
@@ -290,9 +290,9 @@ func (c *RenderCommand) Flags() *flag.Sets {
 		})
 
 		f.BoolVar(&flag.BoolVar{
-			Name:    "render-aux-files",
-			Target:  &c.renderAuxFiles,
-			Default: true,
+			Name:    "no-render-aux-files",
```

Comment:
```suggestion
			Name:    "skip-aux-files",
```

`--no-render-aux-files` is quite a mouthful 😅 

The command is also `nomad-pack render` already, so that part seems redundant. I also wonder if we need a CLI flag at all? When would we want to _not_ render these files?

Reply:
I guess my idea behind having that flag was to allow users to preserve previous behavior?